csrc/scheduler/registry.cpp

```diff
@@ -948,13 +978,17 @@ void HeuristicSummary::validate() const {
           entry_type_map_.count(EntryType::UNROLLABLE_INPUTS_AND_OUTPUTS));
       break;
     }
-    case ScheduleHeuristic::Persistent: {
+    case ScheduleHeuristic::InnerPersistent:
+    case ScheduleHeuristic::OuterPersistent:
+      NVF_ERROR(
+          entry_type_map_.count(EntryType::UNROLLABLE_INPUTS_AND_OUTPUTS));
```

Comment:
not used or cached by innerOuter scheduler.

Reply:
Previously, `EntryType::UNROLLABLE_INPUTS_AND_OUTPUTS` was added to cache for all persistent schedulers.